files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md

```diff
@@ -291,10 +291,10 @@ curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location
 输出应该是这样的 (`curl` 首先会输出一些下载计数器之类的东西):
 
 ```bash
-location: /en-US/docs/Web/API/fetch
-location: /en-US/docs/Web/API/GlobalFetch/GlobalFetch.fetch()
-location: /en-US/docs/Web/API/GlobalFetch/fetch
-location: /en-US/docs/Web/API/fetch
+location: /zh-CN/docs/Web/API/fetch
+location: /zh-CN/docs/Web/API/GlobalFetch/GlobalFetch.fetch()
+location: /zh-CN/docs/Web/API/GlobalFetch/fetch
+location: /zh-CN/docs/Web/API/fetch
```

Comment:
This would be:

```suggestion
location: /en-US/docs/Web/API/fetch
```

![image](https://github.com/mdn/translated-content/assets/15844309/12eeb53e-930a-4be5-b7d8-10934dcb5ad8)

Maybe we should ignore `code` blocks